src/test/resources/data/published_requests/outgoing_mocked_request4_null_or_empty_values.json

```diff
@@ -285,7 +312,16 @@
           "namespace": "investigatorId",
           "value": "01-XXXXXXXa"
         }
-      ]
+      ],
+      "additionalProperties": {
```

Comment:
Suggestion: Make one of the fields in `additionalProperties` null, empty or "null"